docs/user-guide/08-compiling.md

```diff
@@ -179,6 +179,13 @@ we can compile the `computeMain()` entry point to SPIR-V using the following com
 slangc hello-world.slang -target spirv -o hello-world.spv
 ```
 
+> #### Note ####
+> The `slangc` CLI [does not currently support](https://github.com/shader-slang/slang/issues/5541) `[shader(...)]` attributes. Complile the above example with the following command instead:
+> 
+> ```bat
+> slangc hello-world.slang -target spirv -entry computeMain -o hello-world.spv
```

Comment:
This is not completely true though. `slangc hello-world.slang -target spirv` should work just fine. You need to specify the entry point explicitly only when targeting HLSL and GLSL, but not SPIRV.

Reply:
+1, I use this feature regularly. Might be good to clarify what does or doesn't work, but slangc does support at least some mixed entry point work. 

I thought at one point we discussed supporting mixed entry points via partially-compiled slang modules too. I haven't really investigated this though.

Reply:
Updated.